manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
Correct, but kafo won't actually execute the code so that's not a problem. See https://github.com/theforeman/puppet-foreman_proxy/pull/121 for the discussion.